src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+public class Main {
+
+    public static void main(String[] args) {
+        Figure[] figures = new Figure[6];
+        FigureSupplier figureSupplier = new FigureSupplier();
+        for (int i = 0; i < 6; i++) {
+            figures[i] = figureSupplier.getRandomFigure();
```

Comment:
In the main() method we need to create an array of figures (the size of array can be 3 or 6, it doesn't matter). **_The first half of figures in this array should be generated with random parameters_**.

Reply:
Not fixed.
Here you need to check if this is the first half of array or not (no matter how many figures will be in array)